pkgs/data/fonts/junicode/default.nix

```diff
@@ -1,24 +1,19 @@
-{ lib, stdenvNoCC, fetchFromGitHub }:
+{ lib, stdenvNoCC, fetchzip }:
 
-stdenvNoCC.mkDerivation {
+stdenvNoCC.mkDerivation rec {
   pname = "junicode";
-  version = "1.003";
+  version = "2.001";
 
-  src = fetchFromGitHub {
-    owner = "psb1558";
-    repo = "Junicode-font";
-    rev = "55d816d91a5e19795d9b66edec478379ee2b9ddb";
-    hash = "sha256-eTiMgI8prnpR4H6sqKRaB3Gcnt4C5QWZalRajWW49G4=";
+  src = fetchzip {
+    url = "https://github.com/psb1558/Junicode-font/releases/download/v${version}/Junicode_${version}.zip";
+    hash = "sha256-AEbRU0KiB9gLdxcXn8Mn/XLjBEm0qa4cV6+xE1HKU5s=";
   };
 
   installPhase = ''
-    runHook preInstall
-
-    local out_ttf=$out/share/fonts/junicode-ttf
-    mkdir -p $out_ttf
-    cp legacy/*.ttf $out_ttf
-
-    runHook postInstall
+    install -Dt $out/share/fonts/truetype TTF/*.ttf VAR/*.ttf
+    install -Dt $out/share/fonts/opentype OTF/*.otf
+    install -Dt $out/share/fonts/woff2 WOFF2/*.woff2
+    install -Dt $out/share/doc/${pname}-${version} docs/*.pdf
```

Comment:
Maybe the docs could be part of `$doc` instead?

Reply:
That does seem better, yes. I blindly copied this part from other font derivations, most of which seem to put documentation in `$out`, but probably they are the ones in the need of refactoring. I’m not even sure if this documentation actually ends up anywhere in a NixOS install, as `fonts.fonts` seems to ignore it.